### PR TITLE
Improvements to the downloader's command line interface

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/651)

### CODE REVIEW

File: utils/build.gradle [hunk 1]
```diff
@@ -15,4 +15,5 @@ dependencies {
 
     compile "commons-codec:commons-codec:$commonsCodecVersion"
     compile "org.apache.commons:commons-compress:$commonsCompressVersion"
+    compile "org.tukaani:xz:$xzVersion"
```
Comment (mnonnenmacher) [inline utils/build.gradle]: Is the 7zip support of commons compress too limitied, or why can't we use it?

Comment (mnonnenmacher) [reply utils/build.gradle]: Is this used at all? I don't see any imports from `org.tukaani` below.

File: downloader/src/main/kotlin/Main.kt [hunk 2]
```diff
@@ -216,9 +234,7 @@ object Main {
                         )
                 )
             } else {
-                // TODO: Allow to specify the VCS revision as a parameter.
-                val vcs = VcsInfo.EMPTY.copy(url = projectUrl!!)
-
+                val vcs = VcsInfo(type = vcsType, url = projectUrl!!, revision = vcsRevision, path = vcsPath)
                 Package.EMPTY.copy(id = dummyId, vcs = vcs, vcsProcessed = vcs)
```
Comment (mnonnenmacher) [inline downloader/src/main/kotlin/Main.kt]: Shouldn't it be `vcsProcessed = vcs.normalize()`?

File: utils/build.gradle (hunk 1 above)
Comment (sschuberth) [reply utils/build.gradle]: The Apache Commons implementation [actually depends on it](https://stackoverflow.com/a/28277555/1127485), but as it's optional if you're not using 7-Zip, they did not add `xz` as a dependency...

File: downloader/src/main/kotlin/Main.kt (hunk 2 above)
Comment (sschuberth) [reply downloader/src/main/kotlin/Main.kt]: Done.

**Pull request merged**